includes/forms.inc

```diff
@@ -554,3 +555,19 @@ function islandora_ip_embargo_hex_validate($element, &$form_state, $form) {
     form_set_error('islandora_ip_embargo_overlay_text_color', t('Color must match hexideciamal format, requires 3 or 6 characters and A-F 0-9 characters apply (E.g. F0A3B1 or F1A).'));
```

Comment:
Hey @JojoVes 😬  i will approve and merge, just wondering if this (real code task!) could get done here also?
Says: " form_set_error('islandora_ip_embargo_overlay_text_color', t('Color must match hexideciamal format, requires 3 or 6 characters and A-F 0-9 characters apply (E.g. F0A3B1 or F1A).'));"
kinda could be "hexadecimal". Just a tought, if you happen to still have the file open...